bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/types/BNeverType.java

```diff
@@ -0,0 +1,39 @@
+/*
+ *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package org.ballerinalang.jvm.types;
+
+/**
+ * {@code BNeverType} represents the type of a {@code Never}.
+ *
+ * @since 2.0.0-preview1
+ */
+public class BNeverType extends BNullType {
+    /**
+     * Create a {@code BNeverType} represents the type of a {@code Never}.
+     *
+     * @param pkg package path
+     */
+    BNeverType(BPackage pkg) {
```

Comment:
Why is this `BPackage` param required? `BNullType`'s constructor accepts two `String` params`

Reply:
Ah sorry, nvm. I was looking at the wrong class.